src/plugins/fs/ntfs.c

```diff
@@ -400,8 +402,23 @@ BDFSNtfsInfo* bd_fs_ntfs_get_info (const gchar *device, GError **error) {
     lines = g_strsplit (output, "\n", 0);
     g_free (output);
     line_p = lines;
+
     /* find the beginning of the (data) section we are interested in */
-    while (line_p && *line_p && !g_str_has_prefix (*line_p, "bytes per volume"))
+    while (line_p && *line_p && !strstr (*line_p, "Cluster Size"))
+        line_p++;
+    if (!line_p || !(*line_p)) {
+        g_set_error (error, BD_FS_ERROR, BD_FS_ERROR_PARSE, "Failed to parse NTFS file system information");
+        g_strfreev (lines);
+        bd_fs_ntfs_info_free (ret);
+        return NULL;
+    }
+
+    /* extract data from something like this: "Cluster Size: 4096" */
+    val_start = strchr (*line_p, ':');
+    val_start++;
+    cluster_size = g_ascii_strtoull (val_start, NULL, 0);
```

Comment:
This should, ideally, do some error checking (and the same applies to `g_ascii_strtoull()` calls below.